pkgs/development/python-modules/rcfile/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, pythonOlder, isPy3k
+, nose }:
+
+buildPythonPackage rec {
+  pname = "rcfile";
+  version = "0.1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ycbbzx3lshdjjin1lnypknhk7iy2vhr1z7g31y2g9l23dzapg91";
+  };
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.